definitions.yaml

```diff
@@ -181,3 +181,17 @@ definitions:
       email:
         type: string
         description: Email address of the user
+
+  # One of the users in the array as returned by getUsers
+  V4GetUsersUser:
```

Comment:
That name is a bit weird. Can we rename that to `V4UserListItem ` (according to `V4OrganizationListItem`)?